manual/manual/cmds/runtime.etex

```diff
@@ -112,8 +112,11 @@ The following environment variables are also consulted:
 \fi
   \begin{options}
   \item[b] (backtrace) Trigger the printing of a stack backtrace
-        when an uncaught exception aborts the program.
-        This option takes no argument.
+        when an uncaught exception aborts the program. This option does not
+        require an argument. If a bytecode program does not have any available
+        FDs when the runtime attempts to load debugging information, then
+        location information will not be displayed. You can force the runtime
+        to load debugging information at program startup by setting "b=2".
```

Comment:
The documentation is not very clear.  Not everyone knows what an "FD" is.  Plus, the backtrace will tell when "b=2" is required.  An example of an alternate wording:
```suggestion
        when an uncaught exception aborts the program. An optional argument
        can be provided: "b=0" turns backtrace printing off; "b=1" is equivalent to "b"
        and turns backtrace printing on; "b=2" turns backtrace printing on and forces
        the runtime system to load debugging information at program startup time
        instead of at backtrace printing time.
```

Reply:
@dra27 could you please agree to reword the documentation, or state that you don't want it changed?  We need to make progress on PRs.